leaf-ui/js/displayAnalysis.js

```diff
@@ -134,21 +132,21 @@ function adjustSliderWidth(maxValue){
  * @param {Number} sliderValue
  *   Current value of the slider
  * @param {Number} currentValueLimit
- * @param {Object} currentAnalysis
- *   Contains data about the analysis that the back end performed
+ * @param {AnalysisResult} currentAnalysis
+ *  an AnalysisResult object that contains data about the analysis that the back end performed
  */
 function updateSliderValues(sliderValue, currentAnalysis){
 
     analysisResult.selectedTimePoint = sliderValue;
 
     var value = sliderValue;
     $('#sliderValue').text(value);
-    sliderObject.sliderValueElement.innerHTML = value + "|" + currentAnalysis.relativeTime[value];
+    sliderObject.sliderValueElement.innerHTML = value + "|" + currentAnalysis.timePointPath[value];
     // Update the analysisRequest current state.
     analysisRequest.currentState = sliderObject.sliderValueElement.innerHTML;
 
-	for (var i = 0; i < currentAnalysis.numOfElements; i++) {
-		var element = currentAnalysis.elements[i];
+    for (var i = 0; i < currentAnalysis.elementList.length; i++) {
```

Comment:
A forEach loop here might be more efficient since you don't actually need the index value of the list